distributed/tensor_parallelism/fsdp_tp_example.py

```diff
@@ -53,41 +56,10 @@
 [0, 8, ..., 8N-8], [1, 9, ..., 8N-7], ..., [7, 15, ..., 8N-1]
 ======================================================================
 
-More details can be seen in the slide:
-https://docs.google.com/presentation/d/17g6WqrO00rP3MsxbRENsPpjrlSkwiA_QB4r93_eB5is/
+More details can be seen in the PyTorch tutorials:
+https://pytorch.org/tutorials/intermediate/TP_tutorial.html
```

Comment:
I'm getting a 404 on this

Reply:
This is not merged yet, would merge it soon https://github.com/pytorch/tutorials/pull/2829